external/storm-kafka/pom.xml

```diff
@@ -129,6 +129,11 @@
             </exclusions>
         </dependency>
         <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>0.8.2.1</version>
+        </dependency>
```

Comment:
we probably want scope provided here so we don't explicitly package kafka client and rely on the one provided.  Since the new api is java it shouldn't be as big deal as with scala though.